packages/amplify-e2e-tests/src/schema-api-directives/tests/key-howTo4.ts

```diff
@@ -0,0 +1,257 @@
+     }) {
+         content
+         username
+         createdAt
+         editor
+         data1
+         data2
+       }
+     }`;
+
+//query1 with no filter
+export const query1 = `
+query SyncComments {
+  syncComments(filter: {and: [{username: {eq: "akz"}}, {createdAt: {gt: "2010-01-01T00:00Z"}}]}) {
+    items {
+      content
+      createdAt
+      editor
+      data1
+      data2
+      editor
+      username
+    }
+  }
+}
+`;
+export const expected_result_query1 = {
+  data: {
+    syncComments: {
+      items: [
+        {
+          content: 'order2',
+          username: 'akz',
+          createdAt: '2018-01-01T01:05:49.129Z',
+          editor: 'user1',
+          data1: 'example3',
+          data2: 'example4',
+        },
+        {
+          content: 'order1',
+          username: 'akz',
+          createdAt: '2019-01-01T01:05:49.129Z',
+          editor: 'user1',
+          data1: 'example1',
+          data2: 'example2',
+        },
+      ],
+    },
+  },
+};
+//query1 with filter
+export const query2 = `
+query SyncComments {
+  syncComments(filter: {and: [{username: {eq: "akz"}}, {createdAt: {gt: "2010-01-01T00:00Z"}}, { content: {eq : "order1"}}]}) {
+    items {
+      content
+      createdAt
+      editor
+      data1
+      data2
+      editor
+      username
+    }
+  }
+}
+`;
+export const expected_result_query2 = {
+  data: {
+    syncComments: {
+      items: [
+        {
+          content: 'order1',
+          username: 'akz',
+          createdAt: '2019-01-01T01:05:49.129Z',
+          editor: 'user1',
+          data1: 'example1',
+          data2: 'example2',
+        },
+      ],
+    },
+  },
+};
+
+//query1 with no PK and all filter
+export const query3 = `
+query SyncComments {
+  syncComments(filter: {and: [ {createdAt : {gt : "2019-01-01T00:00Z"}}]}) {
+    items {
+      content
+      createdAt
+      editor
+      data1
+      data2
+      editor
+      username
+    }
+  }
+}
+`;
+export const expected_result_query3 = {
+  data: {
+    syncComments: {
+      items: [
+        {
+          content: 'order1',
+          username: 'akz',
+          createdAt: '2019-01-01T01:05:49.129Z',
+          editor: 'user1',
+          data1: 'example1',
+          data2: 'example2',
+        },
+      ],
+    },
+  },
+};
+
+//query1 with no PK and "or" filter
+export const query4 = `
+query SyncComments {
+  syncComments(filter: {or: [ {data1 : {gt : "example3"}},{username : {eq : "user1"}}]}) {
+    items {
+      content
+      createdAt
+      editor
+      data1
+      data2
+      editor
+      username
+    }
+  }
+}
+`;
+export const expected_result_query4 = {
+  data: {
+    syncComments: {
+      items: [
+        {
+          content: 'order1',
+          username: 'user1',
+          createdAt: '2015-01-01T01:05:49.129Z',
+          editor: 'akz',
+          data1: 'example1',
+          data2: 'example2',
+        },
+        {
+          content: 'order3',
+          username: 'akz',
```

Comment:
user2 would do :-D no personal info in code

Reply:
Ahh , I was going to change that :D